lib/ui.ts

```diff
@@ -212,6 +212,8 @@ router.get("/", async ctx => {
         )};
         window.permissionSets = ${JSON.stringify(permissionSets)};
       </script>
+      <script src="https://unpkg.com/mithril@2.0.0-rc.8/mithril.js"></script>
```

Comment:
> Last I checked Rollup doesn't have good support for loading JS dynamically so we decided to stick with Webpack for the client side code.

so I add front end dependencies here and  with source map flag  added  the debugging experience is smooth, we can add break point directly in front end typescript code

Reply:
This defeats the purpose of having certain JS modules loaded dynamically. We also can't rely on external dependencies as some users may be running GenieACS a restricted environment with limited or no Internet access.

Reply:
Thank you for keeping this scenario in mind @zaidka. We run our ACS in a very restricted environment with no internet access.